src/main/java/programminglife/ProgrammingLife.java

```diff
@@ -26,16 +29,21 @@
     private static final String TB_DATA = DATA_FOLDER + "real/TB10.gfa";
     private static final String HUMAN_DATA = DATA_FOLDER + "real/chr19.hg38.w115.gfa";
 
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("m:ss.SSS", Locale.getDefault());
+
     public static void main(String[] args) {
         Graph g;
         String graphFile = TEST_DATA;
+        DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
 
         if (args.length > 0) {
             graphFile = args[0];
         }
 
         try {
-            g = Graph.parse(graphFile, true);
+            long startTime = System.nanoTime();
+            g = Graph.parse(graphFile);
+            System.out.println(String.format("Parsing took %s", DATE_FORMAT.format((System.nanoTime() - startTime) / 1000000)));
```

Comment:
I think it would be better to first get the current time, possibly subtract the first time, and only then print it, instead of doing it all in one line.